esp-config/src/generate.rs

```diff
@@ -0,0 +1,138 @@
+use core::fmt::Write;
+use std::{collections::HashMap, env, fs, path::PathBuf};
+
+const TABLE_HEADER: &str = r#"
+| name | description | default value |
+|------|-------------|---------------|
+"#;
+
+#[derive(Copy, Clone, Debug, PartialEq, Eq)]
+pub struct ParseError;
+
+#[derive(Clone, Debug, PartialEq, Eq)]
+pub enum Value {
+    Number(usize),
+    Bool(bool),
+    String(String),
+}
+
+impl Value {
+    fn parse(&mut self, s: &str) -> Result<(), ParseError> {
+        *self = match self {
+            Value::Bool(_) => match s {
+                "false" => Value::Bool(false),
+                _ => Value::Bool(true), // TODO should existance of the key mean true?
```

Comment:
Let's be strict and only accept a few options
- `true`/`false`
- `yes`/`no`
- single letter abbreviations of these
- `default`, maybe?

Otherwise typoing `flase` would mean true and would be rather surprising